.github/workflows/publish.yaml

```diff
@@ -0,0 +1,22 @@
+# A CI configuration to auto-publish pub packages.
+
+name: Publish
+
+on:
+  pull_request:
+    branches: [ main ]
+    types: [opened, synchronize, reopened, labeled, unlabeled]
+  push:
+    tags: [ '[A-z]+-v[0-9]+.[0-9]+.[0-9]+*' ]
+
+jobs:
+  publish:
+    if: ${{ github.repository_owner == 'dart-lang' }}
+    uses: dart-lang/ecosystem/.github/workflows/publish.yaml@main
+    with:
+      write-comments: false
+      sdk: dev
+      environment: pub.dev # Can be any name, this is the convention though.
```

Comment:
I don't think we want to use an environment when publishing (another button to push w/o a clear benefit).

Reply:
Oh is this the thing that requires the extra approval step?